backend/tests/conftest.py

```
@@ -0,0 +1,63 @@
+from sqlalchemy.pool import StaticPool
+
+from app.database import get_db
+from app.models.base import Base
+
+
+@pytest.fixture
+def client():
+    """Test client fixture with in-memory database"""
+    from app.main import app
+
+    # インメモリデータベースを作成（非同期版）
```

Ruff (0.14.8)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)